### DISCUSSION TITLE
lazer keeps freezing in editor 😭

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#17636] [Q&A]
whenever i put my cursor on the button on the left it seems to have a chance to just freeze up, no clue why
my friends who also use arch linux report the same bug
[logs.zip](https://github.com/ppy/osu/files/8405672/logs.zip)
https://youtu.be/T_mneot5xfk

**Comment by peppy** [comment]
it's probably something arch linux related. that distribution has no shortage of issues, but it's going to be up to you to solve this one. there are no errors on osu!'s side, so it's either something SDL, WM or graphics driver level.

**Reply by Axilotl17** [reply]
alright :(
